extensions/amp-mathml/validator-amp-mathml.protoascii

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-mathml
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-html"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-mathml>
+  html_format: AMP
+  tag_name: "AMP-mathml"
+  requires_extension: "amp-mathml"
+  attrs: {
+    name: "data-selected-cat"
+    value_regex_casei: "(oscar|chloe|bella)"
+  }
+  attrs: {
+    name: "formula"
+    mandatory: true
+  }
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: CONTAINER
+  }
+}
```

Comment:
new line at the end of file

Reply:
added. out of curiosity, why is this required? also, why doesn't the linter pick up this as an issue?